packages/jest-config/src/readConfigFileAndSetRootDir.ts

```diff
@@ -89,8 +93,20 @@ const loadTSConfigFile = async (
   configPath: string,
 ): Promise<Config.InitialOptions> => {
   // Get registered TypeScript compiler instance
-  const registeredCompiler = await getRegisteredCompiler();
+  const docblockPragmas = parse(extract(fs.readFileSync(configPath, 'utf8')));
+  const tsLoader = docblockPragmas['jest-config-loader'] || 'ts-node';
+
+  if (Array.isArray(tsLoader)) {
+    throw new TypeError(
```

Comment:
This is not working in any case, even when  passing multiple loaders like this 
`/**@jest-config-loader ts-node esbuild-register*/` it simply returns a string `'ts-node esbuild-register'` instead of returning an array of strings `['ts-node','esbuild-register']`
is this an expected behavior or a bug in `jest-docblock`?

Reply:
that's expected - for it to be an array you'd need to do

```ts
/**
 * @jest-config-loader ts-node
 * @jest-config-loader esbuild-register
 */
```